packages/dev/core/src/Meshes/abstractMesh.ts

```diff
@@ -134,6 +134,38 @@ export interface IMeshDataCache {
     _vertexData?: { [kind: string]: Float32Array };
 }
 
+/**
+ * Data for mesh hotspot computation
+ */
+export class HotSpotQuery {
+    /**
+     * Mesh name
+     */
+    meshName: string;

```

Comment:
`meshName` is not used by `AbstractMesh.getHotSpotToRef`, so I don't think it should be part of the class.